### ISSUE TITLE
missing command taste-update-data-view

### ISSUE THREAD

**Issue by Kynneb** [ESROCOS/gui-vizkit3d_tastelib#4]
I get a weird error trying to build the package:

```
[ 90%] Run build-script.sh for vizkit3d_bodystate_widgets
[INFO] Executing user-defined init script
Updating DataView.aadl
./update_data_view.sh: line 10: taste-update-data-view: command not found
```

obviously `taste-update-data-view` in fact does exist, though. When I try to go inside the bodystate directory and execute `sh build-script.sh` manually, i even get

```
[INFO] Executing user-defined init script
build-script.sh: line 30: source: user_init_pre.sh: file not found
```

although the file of course is within the folder.

Does that look like anything to you?

**Comment by maxime-esa** [comment]
Perhaps you made an update of TASTE and you need to close the terminal and open a new one?

**Comment by RaquelJalvo** [comment]
It is very weird. Can you do a printenv of PATH ? Is it possible that you have overwritten this environment variable?

**Comment by Kynneb** [comment]
I am not quite sure. The second error I could eventually trace back to executing the build-script wrongly in dash instead of bash (by calling sh .build-script.sh instead of bash build-script.sh or simply executing it) So that seems not be related. 

The PATH seems correct - especially since bash auto-complete finds taste-update-data-view as one would expect. 

Since this error occured when trying to build headless with jenkins I tried it manually on the remote machine via ssh. The error did not happen, however the build did not proceed further then 90% either, seemingly stopping at the "converting interfaceview from V2.0 to V1.3" step.

Then I tried again on my local machine and there it worked to build all the models individually but it was pretty slow.

I will let the remote installation run over night - perhaps this was just a matter of time?

I recently updated the docker image from 9.0 to 9.1 - are there any problems known I could have stumbled into?

**Comment by mmunoz-gmv** [comment]
The TASTE scripts require bash. Maybe try to do "sudo rm /bin/sh", then "sudo ln -s /bin/bash /bin/sh".

**Comment by Kynneb** [comment]
Maybe the error has something to do with building headless (i.e. without X support)?

Because the problematic parts (the models taste projects) try to open GUI progress dialogs.

**Comment by maxime-esa** [comment]
yes, the Converting to V1.3 is done by a command line version of the GUI from Ellidiss, and for some reasons it needs X.

(the dash issue also needs to be fixed)
